prov/efa/test/efa_unit_tests.c

```diff
@@ -1,22 +1,37 @@
 #include "efa_unit_tests.h"
 
+static int reset_global_mocks(void **state)
+{
+	g_ibv_send_wr_list = (struct efa_mock_ibv_send_wr_list) {
```

Comment:
should loop through the list and release the memory.

Reply:
Will take a closer look.